### PR TITLE
feat(handlers): add getAvailableVoiceRegions()

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**
Adds `getVoiceRegions()` which returns a list of available voice server regions when creating a guild.
Renames old `getVoiceRegions()` to `getGuildVoiceRegions()`

NOTE: This PR includes breaking changes

**Status**

- [x] Code changes have been tested against the Discord API, or there are no
  code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [x] This PR includes breaking changes (methods removed or renamed,
    parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation,
  README, etc.

### CODE REVIEW

File: src/api/handlers/guild.ts [hunk 1]
```diff
@@ -731,8 +731,13 @@ export function leaveGuild(guildID: string) {
...
 }
 
+/** Returns an array of voice regions that can be used when creating servers. */
+export function getVoiceRegions() {
+  return RequestManager.get(endpoints.VOICE_REGIONS);
+}
+
 /** Returns a list of voice region objects for the guild. Unlike the similar /voice route, this returns VIP servers when the guild is VIP-enabled. */
-export function getVoiceRegions(guildID: string) {
+export function getGuildVoiceRegions(guildID: string) {
```
Comment (ayntee) [inline src/api/handlers/guild.ts:740]: Separate PR for this.

Comment (itohatweb) [reply src/api/handlers/guild.ts:740]: I didn't do a separate PR because the function added actually has the same name.

Comment (Skillz4Killz) [reply src/api/handlers/guild.ts:740]: I don't think this is resolved, this makes this pr breaking. Rename the other one to something different like `getAllVoiceRegions` and in a separate pr for.

OOOOR

keep the 1 function but make guildID optional.

if (!guildID) get them all

**Review by Skillz4Killz — CHANGES_REQUESTED** [review]

File: src/api/handlers/guild.ts (hunk 1 above)
Comment (itohatweb) [reply src/api/handlers/guild.ts:740]: if the guild is a vip you get all. `getVoiceRegions` only returns available ones so no vip voice servers.

Comment (ayntee) [reply src/api/handlers/guild.ts:740]: So you should create a separate PR changing the name of this function.

Comment (itohatweb) [reply src/api/handlers/guild.ts:740]: How? The added function literally has the same name. If you have a better name I'll change it.

Comment (Skillz4Killz) [reply src/api/handlers/guild.ts:740]: getAllAvailableVoiceRegions

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**